ui/desktop/electron-app/src/ipc/handlers.js

```diff
@@ -194,6 +194,26 @@ handle('isClientAgentRunning', async () => {
   }
 });
 
+/**
+ * Gets boundary cli version
+ */
+handle('getCliVersion', async () => boundaryCli.version());
+
+/**
+ * Gets Cache Daemon version
+ */
+handle('getCacheDaemonVersion', async () => {
```

Comment:
Shouldn't we be returning the full status instead of just the version here and have the app handle parsing out the version? We likely will need more later

Reply:
Agreed, this could be `getCacheStatus` and then the model of an specific domain can parse out what's not need.

Reply:
thanks, updated to receive full status from cache